apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.js

```diff
@@ -31,48 +34,9 @@ function getTourAssets( key ) {
 	return tourAssets[ key ];
 }
 
-const referenceElements = [
-	{
-		desktop: null,
-		mobile: null,
-	},
-	{
-		desktop: null,
-		mobile: null,
-	},
-	{
-		mobile:
-			'.edit-post-header .edit-post-header__toolbar .components-button.edit-post-header-toolbar__inserter-toggle',
-		desktop:
-			'.edit-post-header .edit-post-header__toolbar .components-button.edit-post-header-toolbar__inserter-toggle',
-	},
-	{
-		desktop: null,
-		mobile: null,
-	},
-	{
-		mobile:
-			'.edit-post-header .edit-post-header__settings .interface-pinned-items > button:nth-child(1)',
-		desktop:
-			'.edit-post-header .edit-post-header__settings .interface-pinned-items > button:nth-child(1)',
-	},
-	{
-		desktop: '.edit-post-header .edit-post-header__toolbar .components-button.editor-history__undo',
-	},
-	{
-		mobile: null,
-		desktop: null,
-	},
-	{
-		desktop: null,
-		mobile: null,
-	},
-];
-
 function getTourSteps( localeSlug, referencePositioning ) {
 	return [
 		{
-			referenceElements: referencePositioning && referenceElements[ 0 ],
```

Comment:
Anyway, we could decide to leave `referenceElements: null` or remove it alltogether

Reply:
`referenceElements` should be optional, so we can remove if none needed. This is the type I have on my end right now:

```
export interface Step {
	referenceElements?: {
		desktop?: string;
		mobile?: string;
	};
	meta: {
		[ key: string ]: unknown;
		// | React.FunctionComponent< Record< string, unknown > >
		// | HTMLElement
		// | string
		// | ...
	};
	options?: {
		className?: string;
	};
}
```